products/compute/terraform.yaml

```diff
@@ -190,6 +190,18 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         set_hash_func: 'resourceGoogleComputeBackendServiceBackendHash'
       backends.group: !ruby/object:Overrides::Terraform::PropertyOverride
         diff_suppress_func: 'compareSelfLinkRelativePaths'
+      backends.maxConnections: !ruby/object:Overrides::Terraform::PropertyOverride
+        send_empty_value: true
```

Comment:
Ugh we have precedent for putting these in both api.yaml and terraform.yaml, but I think api.yaml is a bit more common.